core/domain/feedback_services.py

```diff
@@ -143,6 +146,9 @@ def create_message(
     msg.received_via_email = received_via_email
     msg.put()
 
+    # Update the message count in the thread.
+    increase_message_count_by_one(full_thread_id)
```

Comment:
I don't see the point of a separate get and put here. You're updating the thread anyway (see below) so why not just modify the message count prior to the already-existing put()?